pkg/apis/garden/validation/validation_test.go

```diff
@@ -5626,6 +5627,29 @@ var _ = Describe("validation", func() {
 			})
 		})
 
+		Context("KubeControllerManager configuration validation", func() {
+			It("should fail when nodeCIDRMaskSize is out of upper boundary", func() {
+				shoot.Spec.Kubernetes.KubeControllerManager.NodeCIDRMaskSize = makeIntPointer(32)
+
+				errorList := ValidateShoot(shoot)
+				Expect(errorList).ToNot(BeEmpty())
```

Comment:
Can you also check that really this field is the problematic one (like we do in the other tests)?